evap/staff/views.py

```diff
@@ -524,6 +524,12 @@ def course_email(request, semester_id, course_id):
     course = get_object_or_404(Course, id=course_id)
     form = CourseEmailForm(request.POST or None, instance=course)
 
+    if 'export' in request.POST and 'recipients' in request.POST:
+        form.recipient_groups = request.POST['recipients']
+        email_addresses = ", ".join(form.email_addresses())
+        form2 = CourseEmailForm(None, instance=course)
+        return render(request, "staff_course_email.html", dict(semester=semester, course=course, form=form2, addresses=email_addresses))
```

Comment:
did you try using the original `form` here, did that work? because as i understand it, if the user would use the export as "show me the emailadresses before i hit send", now his selection of recipients would be reset?